backend/cal.py

```
@@ -0,0 +1,53 @@
+import os, sys
+
+def calculate_area(radius):
+    area = 3.14 * radius ^ 2
+    return area
+
+def print_area(radius):
+    area = calculate_area(radius)
+    print("The area is: " + area)
+
+def read_file(file_path):
+    file = open(file_path)
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)